src/fidgets/ui/channel.tsx

```
@@ -0,0 +1,96 @@
+    enabled: !!name,
+    queryFn: async () => {
+      const { data } = await axiosBackend.get<{ channel: Channel }>(
+        "/api/farcaster/neynar/channel",
+        { params: { id: name } },
+      );
+      return data.channel;
+    },
+  });
+};
+
+const Channel: React.FC<FidgetArgs<ChannelFidgetSettings>> = ({ settings }) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Channel'. Consider to delete it or rename it.

'Channel' is defined here:


(lint/suspicious/noRedeclare)